app/views/cases/offender_sar/_case_details.html.slim

```diff
@@ -9,6 +9,11 @@
       tbody.sar-basic-details
         = render partial: 'cases/shared/case_type', locals: { case_details: case_details }
         = render partial: 'cases/sar/data_subject', locals: { case_details: case_details }
+        = render partial: 'cases/offender_sar/prison_number', locals: { case_details: case_details }
+        = render partial: 'cases/offender_sar/subject_aliases', locals: { case_details: case_details }
+        = render partial: 'cases/offender_sar/previous_case_numbers', locals: { case_details: case_details }
+        = render partial: 'cases/offender_sar/other_subject_ids', locals: { case_details: case_details }
+        = render partial: 'cases/offender_sar/date_of_birth', locals: { case_details: case_details }
         = render partial: 'cases/sar/subject_type', locals: { case_details: case_details }
         = render partial: 'cases/sar/third_party', locals: { case_details: case_details }
         = render partial: 'cases/shared/requester_name', locals: { case_details: case_details }
```

Comment:
For some reason this looks really nice. I did notice that on Line 9, the HTML class name is still `sar-basic-details` but technically this is a SAR so that's ok right!?

Looking further below at the partials, they are all 1 line partials, is there any point breaking up these offender SAR table rows into distinct partials, rather than just 1 partial called something like `offender_sar_details` or something? Actually are partials even required if this view is only ever used by Offender SARs?

Reply:
All this stuff is as-is from existing code - just following the pattern already set

Reply:
Ditto :)